reservations/allocation_models.py

```diff
@@ -147,8 +147,10 @@ def __init__(
         period_end: datetime.date,
         baskets: Dict[int, List[AllocationBasket]],
         spaces: Dict[int, List[AllocationSpace]],
+        included_baskets: [int] = [],
```

Comment:
I'd prefer naming this to `included_basket_ids` or something like `output_basket_ids`

Reply:
Yeah, sounds better.